travel.py

```diff
@@ -631,7 +631,7 @@ def updateflights(tenant, username):
 class HotelView(SwaggerView):
     """Class for storing Hotel search related information"""
 
-    @api.route('/hotels/<description>/<location>', methods=['GET'])
+    @api.route('/hotels/<description>/<location>/', methods=['GET'])
```

Comment:
Seems like we want the `/` at the end when calling from the FE, so I'm putting this back - I wasn't sure why we needed it, but in the spirit of not breaking things with FE I will leave it for now.